Voeg regel toe voor het altijd teruggeven van objecten (#354)
Logius-standaarden/API-Design-Rules: sections/designRules.md

   </dl>
</div>

<div class="rule" id="/core/always-return-objects" data-type="technical">
   <p class="rulelab">Always return a response with an object</p>
   <dl>
      <dt>Statement</dt>
      <dd>

>          <p>An API MUST return a response with a top-level object, regardless of request method.

Reviewer: Alleen als het een (JSON/XML) body heeft.

>          <p>An API MUST return a response with a top-level object, regardless of request method.

Author: ```suggestion
         <p>A JSON or XML response MUST have a top-level object, regardless of request method.
```

>          For <a>collection resources</a>, the object MUST contain a field (its key MAY be named <code>items</code>) with its value an array of items from that collection.

Reviewer: Past die MAY in de Statement of is het slechts een voorbeeld?

Author: De indruk die ik kreeg van de werksessie was dat er een sterke voorkeur was voor `items`, maar dat we dit niet kunnen afdwingen. Dus het is meer dan een voorbeeld, maar geen vereiste.
